tests/systemtests/client_test.go

```diff
@@ -0,0 +1,330 @@
+//go:build system_test
+
+package systemtests
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/tidwall/gjson"
+
+	"github.com/cosmos/cosmos-sdk/client/grpc/cmtservice"
+	"github.com/cosmos/cosmos-sdk/codec"
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
+	"github.com/cosmos/cosmos-sdk/testutil"
+	qtypes "github.com/cosmos/cosmos-sdk/types/query"
+)
+
+func TestQueryNodeInfo(t *testing.T) {
+	baseurl := fmt.Sprintf("http://localhost:%d", apiPortStart)
+	sut.ResetChain(t)
+	sut.StartChain(t)
+
+	qc := cmtservice.NewServiceClient(sut.RPCClient(t))
+	res, err := qc.GetNodeInfo(context.Background(), &cmtservice.GetNodeInfoRequest{})
+	assert.NoError(t, err)
+
+	v := NewCLIWrapper(t, sut, true).Version()
+	assert.Equal(t, res.ApplicationVersion.Version, v)
+
+	restRes, err := testutil.GetRequest(fmt.Sprintf("%s/cosmos/base/tendermint/v1beta1/node_info", baseurl))
```

Comment:
For later, we should be adding a way to distinguish a v2 app with comet and one without.
Nothing to do here, but eventually we should skip some v2 system depending on the consensus engine we want to test

Reply:
I'll add a todo